developer-docs-site/docs/index.md

```diff
@@ -0,0 +1,142 @@
+  <div class="col">
+    <div class="card card-body h-100 d-flex flex-column">
+    <a href="concepts/aptos-concepts" class="card-title card-link stretched-link"> <h2>Concepts</h2></a>
+    <p class="card-text">Learn the key concepts of the Aptos Blockchain, including transactions, accounts, gas and transaction fees, nodes, and more. </p>
+</div>
+</div>
+  <div class="col">
+    <div class="card card-body h-100 d-flex flex-column" >
+    <a href="guides/aptos-guides" class="card-title card-link stretched-link"> <h2>How it works</h2></a>
+    <p class="card-text">How does interacting with the Aptos Blockchain work, how Move works on Aptos, how to create a signed transaction, and more.</p>
+</div>
+  </div>
+  <div class="col">
+    <div class="card card-body h-100 d-flex flex-column">
+    <a href="guides/move-guides/move-on-aptos" class="card-title card-link stretched-link"> <h2>Move on Aptos</h2></a>
+    <p class="card-text">Learn how Move works on the Aptos Blockchain.</p>
+</div>
+</div>
+</div>
+</div>
+
+## Nodes
+
+<div class="docs-card-container">
+<div class="row row-cols-1 row-cols-md-2a g-4">
+  <div class="col">
+    <div class="card card-body h-100 d-flex flex-column" >
+    <a href="nodes/nodes-index" class="card-title card-link stretched-link"> <h2>Run a Validator or a FullNode</h2></a>
+    <p class="card-text">Install and run a Validator or a FullNode.</p>
+</div>
+</div>
+  <div class="col">
+    <div class="card card-body h-100 d-flex flex-column"  >
+    <a href="nodes/nodes-index#for-a-local-testnet" class="card-title card-link stretched-link"> <h2>Run a local testnet</h2></a>
+    <p class="card-text">Run a Validator node on your local testnet.</p>
+</div>
+  </div>
+  <div class="col">
+    <div class="card card-body h-100 d-flex flex-column"  >
+    <a href="nodes/nodes-index#staking" class="card-title card-link stretched-link"> <h2>Stake on Aptos</h2></a>
+    <p class="card-text">Stake your Aptos coins on the Aptos testnet.</p>
+</div>
+</div>
+</div>
+</div>
+
+## SDKs and APIs
+
+<div class="docs-card-container">
+<div class="row row-cols-1 row-cols-md-3a g-4">
+  <div class="col">
+    <div class="card h-100" >
+    <div class="card-body d-flex flex-column" >
+    <a href="https://aptos-labs.github.io/ts-sdk-doc/" class="card-title card-link stretched-link"> <h2>Typescript SDK</h2></a>
+    <p class="card-text">API reference for transacting with the Aptos Blockchain.</p>
+</div>
+</div>
+</div>
+  <div class="col">
+  <div class="card h-100" >
+    <div class="card-body d-flex flex-column"  >
+    <a href="/rest-api" class="card-title card-link stretched-link"> <h2>REST API</h2></a>
+    <p class="card-text">Reference for the REST API to interact with the Aptos Blockchain.</p>
+</div>
+</div>
+</div>
+</div>
+</div>
+
+## Aptos devnet
+
+:::info Aptos Devnet Resets
+The Aptos devnet is reset every Thursday Morning (PST). See the latest updates in [Aptos Discord][discord].
```

Comment:
Should we mention the exact time here? Devnet releases can skip a week or are done not on Thursday sometimes.